src/components/Link.js

```diff
@@ -12,15 +12,14 @@ import { WrapperRow } from './Wrapper';
 export const Link = ({ url, description, openWebScreen }) => (
   <TouchableOpacity
     onPress={() => openLink(url, openWebScreen)}
-    accessibilityLabel={`${description} (Taste)`}
+    accessibilityLabel={`(${description}) Link`}
```

Comment:
You refactored other strings into consts, which have parentheses arount them, so you should probably do it here as well

Reply:
I haven't refactored link into consts because it doesn't make sense to me to have in consts something like this : 
`link : 'Link'`  . 
Or do you mean that I should add round brackets around Link ?

Reply:
solved in 71c498a

Reply:
:+1: